CHANGELOG.md

```diff
@@ -156,6 +156,8 @@ Alternate implementation which directly queries from middleware using view call
 
 * Removed `ethers` and `ethers-signers` from the root `Cargo.toml` in [#551](https://github.com/Layr-Labs/eigensdk-rs/pull/551).
 
+* Removed the `eigen-logging` crate and adopted the `tracing` crate as the standard logging in [#552](https://github.com/Layr-Labs/eigensdk-rs/pull/552).
```

Comment:
Let's add an explanation on how to migrate here, with links to the `tracing` crate docs.

Reply:
LGTM. Left a comment:

https://github.com/Layr-Labs/eigensdk-rs/pull/552/commits/0e8198272fc091e34e758bc0ffd263f84cd3bb6a#r2143143701